test/runtests.jl

```diff
@@ -1,3 +1,6 @@
+import Pkg
+Pkg.pkg"add JuMP#master Statistics"
```

Comment:
I don't understand why `Statistics` was necessary. Never seen this problem before...